deluge/ui/gtkui/connectionmanager.py

```diff
@@ -233,6 +234,23 @@ def add_host(self, host, port, username='', password=''):
         # Update the status of the hosts
         self.__update_list()
 
+    def on_entry_host_paste_clipboard(self, widget):
+        paste = gtk.clipboard_get().wait_for_text()
```

Comment:
Well, it is the pasted text and not the clipboard returned by clipboard_get(), but ok.